ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -3284,6 +3308,41 @@ def test_jax_trunc(
     )
 
 
+@handle_frontend_test(
+    fn_tree="jax.numpy.unwrap",
+    dtype_x_axis=_get_dtype_input_vector_axis(),
+    discont=st.floats(min_value=0, max_value=3.0),
+    period=st.floats(min_value=2 * np.pi, max_value=10.0),
+    test_with_out=st.just(False),
+)
+def test_jax_unwrap(
+    *,
+    dtype_x_axis,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+    discont,
+    period,
+):
+    dtype, x, axis = dtype_x_axis
+    assume(not np.any(np.isclose(x[0], 0)))
```

Comment:
This should be handled while generating the values, so you could remove this line.

Reply:
@bipinKrishnan - Done this has been removed now.